sigstore/_cli.py

```diff
@@ -427,12 +412,21 @@ def _sign(args: argparse.Namespace) -> None:
     elif args.fulcio_url == DEFAULT_FULCIO_URL and args.rekor_url == DEFAULT_REKOR_URL:
         signer = Signer.production()
     else:
-        ct_keyring = CTKeyring([load_pem_public_key(args.ctfe_pem.read())])
+        # Assume "production" keys if none are given as arguments
+        updater = TrustUpdater.production()
+        if args.ctfe_pem is not None:
+            ctfe_keys = [args.ctfe_pem.read()]
+        else:
+            ctfe_keys = updater.get_ctfe_keys()
```

Comment:
If we end up doing this for a custom Fulcio URL, it'd be good to log a warning as this is probably not what the user intended.

This would partially address #324.

I can follow up with another PR if you feel this is out of scope.

Reply:
That sounds like a good idea but since there's even an issue already, let's leave it for a future PR

Reply:
Yep, sounds good.